manpages/brew.1

```diff
@@ -119,7 +119,7 @@ Show what would be removed, but do not actually remove anything\.
 .
 .TP
 \fB\-s\fR
-Scrub the cache, including downloads for even the latest versions\. Note that downloads for any installed formulae or casks will still not be deleted\. If you want to delete those too: \fBrm \-rf "$(brew \-\-cache)"\fR
+Scrub the cache, including downloads for even the latest versions\. Note that downloads for any installed formulae or casks will still not be deleted\. If you want to delete those too: \fBrm \-rf "$(brew?\-\-cache)"\fR
```

Comment:
It seems like there's a small problem here – there's a "?" instead of a space, perhaps because the nbsp isn't recognised during manpage generation. I was able to see the "?" locally by checking out this PR branch and running `man brew`.